src/config/units.yaml

```
@@ -18,43 +18,116 @@ duration:
   description: refers to the duration of the observation
   unit: seconds
   aggregation: sum
-timestamp:
-  description: refers to the time of occurrence of the observation
-  unit: RFC3339
-  aggregation: None
-carbon:
-  description: an amount of carbon emitted into the atmosphere
-  unit: gCO2e
-  aggregation: sum
-energy:
-  description: amount of energy utilised by the component
-  unit: kWh
-  aggregation: sum
 e-cpu:
   description: Energy consumed by the CPU of the component
   unit: kWh
   aggregation: sum
+el:
+  description: expected lifespan for some component
+  unit: seconds
+  aggregation: none
 e-mem:
   description: Energy consumed by the Memory of the component
   unit: kWh
   aggregation: sum
-e-net:
-  description: Energy consumed by the Network of the component
-  unit: kWh
-  aggregation: sum
 embodied-carbon:
   description: Embodied Emissions of the component
   unit: gCO2e
   aggregation: sum
-operational-carbon:
-  description: Operational Emissions of the component
-  unit: gCO2e
+energy:
+  description: amount of energy utilised by the component
+  unit: kWh
+  aggregation: sum
+e-net:
+  description: Energy consumed by the Network of the component
+  unit: kWh
   aggregation: sum
 expected-lifespan:
   description: Total Expected Lifespan of the Component in Seconds
   unit: seconds
   aggregation: None
+functional_unit:
+  description: the name of the functional unit in which the final SCI value should be expressed, e.g. requests, users
+  unit: none
+  aggregation: none
+functional_unit_duration:
+  description: how many units of functional_unit_time the final SCI value should be expressed in.
+  unit: functional_unit_time
+  aggregation: none
+functional_unit_time:
+  description: string describing the unit of time in which the final SCI calculation should be expressed
+  unit: none
+  aggregation: none
+gpu-util:
+  description: refers to CPU utilization.
+  unit: percentage
+  aggregation: avg
+grid-ci:
```

```suggestion
grid-carbon-intensity:
```

I can't in my head justify keeping grid-ci (which I admit I named) and at the same time expanding some of our other shorter acronyms out.